packages/edit-navigation/src/store/controls.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * WordPress dependencies
+ */
+import { default as triggerApiFetch } from '@wordpress/api-fetch';
+import { createRegistryControl } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { menuItemsQuery } from './utils';
+
+/**
+ * Trigger an API Fetch request.
+ *
+ * @param {Object} request API Fetch Request Object.
+ * @return {Object} control descriptor.
+ */
+export function apiFetch( request ) {
+	return {
+		type: 'API_FETCH',
+		request,
+	};
+}
+
+export function getPendingActions( postId ) {
+	return {
+		type: 'GET_PENDING_ACTIONS',
+		postId,
+	};
+}
+
+export function isProcessingPost( postId ) {
+	return {
+		type: 'IS_PROCESSING_POST',
+		postId,
+	};
+}
+
+export function getMenuItemToClientIdMapping( postId ) {
+	return {
+		type: 'GET_MENU_ITEM_TO_CLIENT_ID_MAPPING',
+		postId,
+	};
+}
+
+export function getNavigationPost( menuId ) {
```

Comment:
`getNavigationPost` seems like the argument is a post id. Would `getNavigationPostByMenuId` be clearer?

Reply:
It should match the name of the selector. I'm not against renaming the selector to something like `getNavigationPostForMenu( menuId )`, though, if you want! 🙂